coderd/database/pubsub/pubsub.go

```diff
@@ -528,6 +566,20 @@ func (p *PGPubsub) Collect(metrics chan<- prometheus.Metric) {
 	p.qMu.Unlock()
 	metrics <- prometheus.MustNewConstMetric(currentSubscribersDesc, prometheus.GaugeValue, float64(subs))
 	metrics <- prometheus.MustNewConstMetric(currentEventsDesc, prometheus.GaugeValue, float64(events))
+
+	// additional metrics
+	ctx, cancel := context.WithTimeout(context.Background(), LatencyMeasureInterval)
```

Comment:
Small quibble with the name: "interval" sounds to me like a periodic thing, so I would suggest `LatencyMeasureTimeout`

Reply:
Good call
Thanks for the patience on this PR!